tools/blast_parser/conf/blast_parser_wrapper.xml

```diff
@@ -0,0 +1,34 @@
+<tool id="blast_parser_wrapper" name="blast_parser" version="0.1.1">
+	<description>
+		Parse 12-column BLAST result into 3-column hcluster_sg input
+	</description>
+	 
+	<command>
+		<![CDATA[
+blast_parser.pl 
```

Comment:
Using `perl $__tool_directory__/blast_parser.pl` is going to be easier when installing from the Tool Shed (no need to modify the PATH).